packages/cozy-sharing/src/SharingProvider.jsx

```diff
@@ -182,7 +183,7 @@ export class SharingProvider extends Component {
       })
     )
     this.setState({ hasLoadedAtLeastOnePage: true })
-    fetchNextPermissions(permissions, this.dispatch, client)
+    await fetchNextPermissions(permissions, this.dispatch, client)
```

Comment:
For the record: https://github.com/cozy/cozy-libs/pull/914#discussion_r367520130 

So I'm wondering if you should not: 
```
fetchNextPermissions().then(() => setState('loaded', true))
```

instead of waiting them here